src/route/AppRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+
+import { Layout } from '../components/Layout/Layout';
+import EmployeesPage from '../pages/EmployeesPage/EmployeesPage';
+import LoginPage from '../pages/LoginPage/LoginPage';
+import SignupPage from '../pages/SignupPage/SignupPage';
+
+export default function AppRoute() {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route path="/signup" element={<SignupPage />} />
+        <Route path="/login" element={<LoginPage />} />
+        <Route path="/employees" element={<EmployeesPage />} />
+        <Route path="*" element={<LoginPage />} />
+      </Route>
+    </Routes>
+  );
+}
```

Comment:
You can add Private and Restricted routes for the app. It looks and works better because they redirect the user. I attach a link and my example
https://github.com/INeizvestnaya/Mini-paint/pull/1/files#diff-6b2aab9604475c52f1a82f457ba2f7a3b3ce5c49381691de250b764897ce78b8
https://jasonwatmore.com/post/2022/06/24/react-router-6-private-route-component-to-restrict-access-to-protected-pages#:~:text=React%20App%20Component%20with%20Protected%20Route&text=The%20%2Flogin%20route%20is%20public,paths%20to%20the%20home%20page.